core/src/components/toggle/toggle.tsx

```diff
@@ -63,6 +64,11 @@ export class Toggle implements ComponentInterface {
    */
   @Prop() value?: string | null = 'on';
 
+  /**
+   * Enables the on/off accessibility switch labels within the toggle.
+   */
+  @Prop() switchLabelsEnabled = false;
```

Comment:
APIs should be present tense according to our style guide.

Maybe something like `enableLabels`, `enableOnOffLabels`, or `enableSwitchLabels`? I am partial to "enableOnOffLabels" as there is no confusion as to whether or not this prop impacts `ion-label`.

Reply:
What do you think about defaulting this to `undefined`? If Capacitor adds the ability to automatically detect the I/O preference in the device settings, we could support that functionality without breaking changes as developers would be able to opt-out of this functionality by explicitly setting `false`. If we default to `false`, then having the feature be enabled by default is harder to do without breaking changes.

I know the design doc states it should default to `false`, so feel free to disregard. Just might make life a bit easier down the road if we support that new functionality.

Reply:
I think defaulting to undefined makes sense here, I'll update the design doc.